scripts/generate_sheet.py

```diff
@@ -9,7 +9,7 @@
 from bugbug.models import get_model_class
 
 
-def generate_sheet(model_name, token):
+def generate_sheet(model_name, token, span, threshold):
```

Comment:
And let's call the variable days instead of span.

Reply:
ok, making all the `span` s into `days`